Marlin/src/pins/esp32/pins_MKS_TINYBEE.h

```diff
@@ -170,11 +170,9 @@
   #define BTN_EN2                    EXP2_05_PIN
 
   #if ENABLED(FYSETC_MINI_12864_2_1)
```

Comment:
You'll notice from the previous LCD migration commits that each migrated display is being placed above and outside of the `HAS_WIRED_LCD` block to ensure that the LCD pins file is defining the whole interface. In leaving this inside of the `HAS_WIRED_LCD` block, the lines just above are defining BEEPER, EN, RS, ENC, EN1, EN2, which is not what we want.

Reply:
If you find exceptions, those are just cases where I haven't finished the migration, sometimes because I still have questions to follow up on and am trying not to break things in the meantime.

Reply:
I totally missed important detail